modules/combined/tests/contact_verification/patch_tests/plane_3/plane3_template1.i

```diff
@@ -270,7 +270,7 @@
   solve_type = 'PJFNK'
 
   petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu     superlu_dist'
+  petsc_options_value = 'lu            superlu_dist'
```

Comment:
We take a predictor-correcter approach to whitespace on the BISON team, eventually converging on the standard?
Can I push whitespace corrections to my branch and still have the correction show up here on Fande's branch?